packages/@icon-magic/cli/src/usage.ts

```diff
@@ -0,0 +1,62 @@
+const commandsDescription = {
+  build: {
+    name: 'build',
+    description:
+      'Construct flavors that an icon from its variants, after applying the build plugins.'
```

Comment:
(same from the MD doc)

Reply:
Do you mean that this description should be the same from the MD doc? Won't that be too descriptive?